### DISCUSSION TITLE
Tag Stop Signs

### DISCUSSION THREAD

**Discussion by catalinlis** [carla-simulator/carla#6900] [General]
I have a project for collecting a dataset from simulation and I need to annotate signs such as Stop, Speed Limits, Turn Right, Turn Left and so on. I can use semantic segmentation camera, but the camera label all the traffic signs with the same color. The question is how can I use a label for each sign in order to differentiate each type of sign. Thank you!

**Comment by catalinlis** [comment]
Update:

After adding new tags as described in this section: https://carla.readthedocs.io/en/latest/tuto_D_create_semantic_tags/ you have to make the following commands:

1. make PythonAPI
2. make LibCarla
3. open cmd in carla\PythonAPI\carla\dist and do the following command pip3 install --force-reinstall carla-0.9.14-cp310-cp310-win_amd64.whl and you will have the update for carla library

**Reply by ruoyaow** [reply]
Hi, thanks for sharing!

After the following steps, is it possible to directly extract specific labeled traffic signs by CityObjectLabel in python? I've done the add tag step, but I'm not quite sure how to move the specific objects under that tag.

**Comment by zaidi007** [comment]
@ruoyaow Hi, i am also trying to label dataset images taken from CARLA simulation with traffic sign type. I am searching from many days, but till now got no solution. Can you please tell me how to do it please.
